SubscribeIT/ngDesk
# Fixed font bug in article

Closes #332 

Test Case 1
Name :- Render article font family
1. Login to ngdesk.
2. Goto -> knowledge base -> click on general -> click on any article.
3. Select the article body and check the font family.
4. Observe no error.

Test Case 2
Name :- Create article tinymce
1. Login to ngdesk.
2. Goto -> knowledge base -> New -> article.
3. Observe the default font is set to roboto.
4. Type something and select the article body and check the font family.
5. Observe no error.
6. Click on font drop down and observe roboto is added.


Screenshots :- 

![Screenshot from 2022-01-07 16-17-24](https://user-images.githubusercontent.com/89504119/148533991-f21a4644-602e-4671-ab6d-7104e06480ea.png)
![Screenshot from 2022-01-07 16-18-20](https://user-images.githubusercontent.com/89504119/148534000-ec371afe-726e-4acf-a96c-b007854f5cd5.png)
